messaginginapp/src/main/java/io/customer/messaginginapp/ui/controller/ModalInAppMessageViewController.kt

```diff
@@ -13,7 +14,8 @@ internal class ModalInAppMessageViewController(
     viewDelegate = viewDelegate,
     platformDelegate = platformDelegate
 ) {
-    private var isMessageDisplayed: Boolean = true
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal var shouldDispatchDisplayEvent: Boolean = true
```

Comment:
the class is `internal`, making the properties internal, if its a style thing thats alright but its not needed.

Reply:
Yup, not needed in already internal class, removed it.